From: Kay Sievers <kay@vrfy.org>
Date: Wed, 4 Jun 2014 17:55:14 +0200
Subject: hwdb: fix case-sensitive match

(cherry picked from commit 4c02dd7153f970244950b5e00f7bdfea8d2ff0be)
(cherry picked from commit 47023e8267dafc845ed51544f193e33de6c2e350)
---
 hwdb/60-keyboard.hwdb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hwdb/60-keyboard.hwdb b/hwdb/60-keyboard.hwdb
index 17f06e82..3071c619 100644
--- a/hwdb/60-keyboard.hwdb
+++ b/hwdb/60-keyboard.hwdb
@@ -868,7 +868,7 @@ keyboard:dmi:bvn*:bvr*:bd*:svnOQO*Inc.*:pnOQO*Model*2*:pvr*
 ###########################################################
 
 # Plantronics .Audio 626 DSP
-keyboard:usb:v047fpC006*
+keyboard:usb:v047FpC006*
  KEYBOARD_KEY_b002f=f20                                # Microphone mute button; should be micmute
 
 ###########################################################
